### ISSUE TITLE
question about finetune batchnorm layer

### ISSUE THREAD

**Issue by clytieai** [tensorflow/models#4292]
If i use 8 pieces of GPUs and each batchsize is 2, are the parametres of BN  finetuned by  using 16 images or just 2 images each time ?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
